black.py

```diff
@@ -2199,20 +2197,20 @@ def right_hand_split(
     body.should_explode = should_explode(body, opening_bracket)
     bracket_split_succeeded_or_raise(head, body, tail)
     if (
-        # the body shouldn't be exploded
+        # The body shouldn't be exploded.
```

Comment:
I know this is subjective but for me these read better in the original style. I read all these comments in this `if` expression as if they were a single huge sentence, but introducing those full stops break this for me at least. I would capitalize the first comment and put a `.` at the end of the last one, but otherwise leave everything as is.

Reply:
Updated. Thanks for the comment!